amplicon-pooling.ipynb

```diff
@@ -1164,11 +1162,12 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 476,
+   "execution_count": 30,
    "metadata": {},
    "outputs": [],
    "source": [
     "if len(sheets):\n",
+    "    # TODO: Shouldn't all sheets be validated before merging?\n",
```

Comment:
I agree, just opened a ticket. I would suggest using comments in the PR or opening tickets for these notes :+1:.

```suggestion
```